build/install failures due to yanked crate and also with Rust 1.53.0

Our CI job just started failing to run `cargo install cargo-generate-rpm --version 0.4.0`. This has been working fine until now. It fails due to a yanked aesni crate version, see https://github.com/NLnetLabs/routinator/runs/3171059667#step:9:17:

```
Caused by:
  failed to select a version for the requirement `aesni = "^0.7"`
  candidate versions found which didn't match: 0.99.99, 0.10.0, 0.9.0, ...
```

Running with `--locked` fails for a different reason:

```
cargo install cargo-generate-rpm --version 0.4.0 --locked
    Updating crates.io index
  Installing cargo-generate-rpm v0.4.0
warning: package `aes-soft v0.4.0` in Cargo.lock is yanked in registry `crates.io`, consider running without --locked
warning: package `aesni v0.7.0` in Cargo.lock is yanked in registry `crates.io`, consider running without --locked
   Compiling autocfg v1.0.1
...
   Compiling rand_core v0.5.1
error[E0308]: mismatched types
  --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/lexical-core-0.7.4/src/atof/algorithm/bhcomp.rs:62:24
   |
62 |     let bytes = bits / Limb::BITS;
   |                        ^^^^^^^^^^ expected `usize`, found `u32`
```

**Works with Rust older than 1.53.0:**

One final note: Both my local machine and our CI build both used Rust 1.53.0. When I tried `cargo build` locally using Rust 1.47.0 and Rust 1.52.0, both succeeded!

In a clean Docker Ubuntu 18:04 container the following worked for me:

```
# rustup toolchain install 1.52.0
# cargo +1.52.0 install cargo-generate-rpm --version 0.4.0 --locked
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -16,7 +16,7 @@ edition = "2018"
 [dependencies]
 rpm-rs = { git = "https://github.com/Richterrettich/rpm-rs.git", rev = "2dac707469cdc3a6c7032abd98d13bf494e00665" }
 toml = "0.5"
-cargo_toml = "0.8"
+cargo_toml = "0.9"
 getopts = "0.2"
 thiserror = "1"
 elf = "0.0.10"

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -14,7 +14,7 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-rpm-rs = "0.6"
+rpm-rs = { git = "https://github.com/Richterrettich/rpm-rs.git", rev = "2dac707469cdc3a6c7032abd98d13bf494e00665" }
 toml = "0.5"
 cargo_toml = "0.8"
 getopts = "0.2"

